website/docs/reference/widgets/list.md

```diff
@@ -150,97 +51,77 @@ You’ll see that in the JSON snippet below, there is a collection of books with
 
-<VideoEmbed host="youtube" videoId="iJICPdtoUQc" title="Visible" caption="Visible"/>
+| Property                   | Description                                                                                                                                                                                                                                               |
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Items**                  | Allows you to bind static or dynamic data collection to the widget.                                                                                                                                                                                       |
+| **Server-side Pagination** | Enables you to implement pagination by limiting the number of results fetched per Query request.  |
+| **Visible**                | Controls widget's visibility on the page. When turned off: The widget is visible when the app is published. It appears translucent when in Edit mode.                                                                                            |
+| **Animate Loading**        | When turned off, the widget loads without any skeletal animation. You can use a toggle switch to turn it on/off. You can also turn it off/on using javascript by enabling the JS label next to it.                                                    |
 
 
-### Binding Properties
+### Reference properties
 
 These properties allow you to bind your List widget with any other widget in queries or JS objects. The following table lists all the binding properties.
 
-| Binding Property    | Description                                                                                                                                                                                                                               |
+| Reference Property    | Description                                                                                                                                                                                                                               |
 | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **backgroundColor** | Represents the widget's **Background Color** setting as a CSS `color` value _(string)_.                                                                                                                                                   |
```

Comment:
Both these properties are reference properties as shown in screenshot below. keep them here
<img width="205" alt="Screenshot 2023-02-22 at 14 05 46" src="https://user-images.githubusercontent.com/32227135/220566351-cc02415c-bd9a-46e7-a426-b5a918c6bd5b.png">